factory.go

```diff
@@ -136,6 +138,14 @@ func (f *LoggerFactory) setFormat(l *logrus.Logger) error {
 		f := new(logrus.JSONFormatter)
 		f.TimestampFormat = time.RFC3339Nano
 		l.Formatter = f
+	case "fluentd":
+		f := new(logrus.JSONFormatter)
+		f.FieldMap = logrus.FieldMap{
+			logrus.FieldKeyTime:  "timestamp",
```

Comment:
Maybe we always want to use `timestamp` by default?

cc @rporres @mcuadros

Reply:
we're using now `time`, but let me know if you prefer `timestamp` also for json